### ISSUE TITLE
Add softmax for predictions

### ISSUE THREAD

**Issue by benjaminjellis** [benjaminjellis/APPLE#75]
Something like 
`result = self.net(features_tensor)`
`sm = nn.Softmax(dim = 1)`
`result = sm(result)`
`_, predicted = torch.max(result.data, 1)`
`result = result.detach().numpy()`

**Comment by benjaminjellis** [comment]
Fixed in commit 94ef728
